Owncloud iOs Client/Files/Edit/EditFileViewController.m

```diff
@@ -191,14 +246,24 @@ - (void) sendTextFileToUploadsByTempLocalPath:(NSString *)tempLocalPath andFileN
         upload.estimateLength = (long)fileLength;
         upload.userId = self.currentFileDto.userId;
         upload.isLastUploadFileOfThisArray = YES;
-        upload.status = pendingToBeCheck;
+        if (self.isModeEditing) {
```

Comment:
Maybe you could include the below the "if (self.isModeEditing)" code in this "if/else" condition